.github/CODEOWNERS

```diff
@@ -428,3 +428,5 @@ Makefile                                              @thiagokokada
 /modules/programs/ion.nix                             @jo1gi
 
 /modules/services/plex-mpv-shim.nix                   @starcraft66
+
+/modules/programs/zellij.nix                          @mainrs
```

Comment:
This should go up with the other `/modules/programs`. Not sure why ion is down here, but it shouldn't be either.

Reply:
I don't think I read somewhere that it had to be alphabetically sorted. Might be worth doing that once in a separate PR and updating `CONTRIBUTING`. I might have missed it though.